entrofy/mappers.py

```diff
@@ -83,11 +84,18 @@ def transform(self, column):
         nonnulls = ~column.isnull()
 
         for key in self._map:
-            df[key][nonnulls] = column[nonnulls].apply(self._map[key])
-            df[key][~nonnulls] = None
+            df['{}{}'.format(self.prefix, key)][nonnulls] = column[nonnulls].apply(self._map[key])
+            df['{}{}'.format(self.prefix, key)][~nonnulls] = None
 
         return df
 
+    def _prepend_prefix(self, targets, prefix):
+        new_targets = {}
+        for key, t in six.iteritems(targets):
+            new_key = '{}{}'.format(prefix, key)
+            new_targets[new_key] = t
+
+        return new_targets
```

Comment:
Should `_prepend_prefix` use `self.prefix` instead of a separate parameter `prefix`?